### PR TITLE
even more final changes to clang style

### PR DESCRIPTION
**Pull request by rwcarlsen**
@dschwen - this is your other last chance to make style change suggestions ;-)

### CODE REVIEW

**Comment by permcody** [comment]
@dschwen - please merge if you are reasonably satisfied.

File: .gitattributes [hunk 1]
```diff
@@ -20,3 +20,11 @@ contrib/* linguist-vendored
...
 docs/media/**/*.gif filter=lfs diff=lfs merge=lfs -text
 docs/media/**/*.svg filter=lfs diff=lfs merge=lfs -text
+
+# TODO(rwcarlsen): Remove this after a while.
+framework/include/**/*.h merge=clang-format
+framework/src/**/*.C     merge=clang-format
+test/include/**/*.h      merge=clang-format
+test/src/**/*.C          merge=clang-format
+modules/**/include/**/*.h   merge=clang-format
+modules/**/src/**/*.C       merge=clang-format
```
Comment (dschwen) [inline .gitattributes]: This will not cause issues _before_ the big reformat patch is applied?

Comment (dschwen) [inline .gitattributes]: I guess not...

**Review by dschwen — APPROVED** [review]

File: .gitattributes (hunk 1 above)
Comment (rwcarlsen) [inline .gitattributes]: No - the user has to set it up in their git config and put the merge driver in their path for this to do anything.

**Pull request merged**